charts/karpenter/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: karpenter
+spec:
+  minAvailable: 1
```

Comment:
Given we default to just 1 replica, I think this will cause problems when evicting Karpenter pods.  We will always violate the PDB and have to delete the pod manually.

Reply:
Perhaps we could do something like this?

```
{{ if eq .Values.replicas 1 }} 
minAvailable: 0
{{ else }}
minAvailable: {{ .Values.podDisruptionBudget.minAvailable }}
{{ end }}
```
(not sure if that's totally correct)

Reply:
If we included topologySpread like in this PR https://github.com/aws/karpenter/pull/1427, we could increment the controller replicas to 2